mindee/product/invoice_splitter/invoice_splitter_v1_document.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+
+from mindee.parsing.common import StringDict, clean_out_string
+from mindee.parsing.common.prediction import Prediction
+from mindee.product.invoice_splitter.invoice_splitter_v1_page_group import (
+    InvoiceSplitterV1PageGroup,
+)
+
+
+class InvoiceSplitterV1Document(Prediction):
+    """Document data for Invoice Splitter, API version 1."""
+
+    invoice_page_groups: List[InvoiceSplitterV1PageGroup] = []
```

Comment:
NEVER init a list or dict at the class level, **the value will be shared across instances**!

Reply:
:scream: